Makefile

```diff
@@ -0,0 +1,4 @@
+check:
+	go build -o main && rm main
```

Comment:
Esto construye un main, no comprueba la sintaxis. Os dije además que no pusiérais ningún main. Pero si no lo ponéis, esta no es la regla correcta para compilar; tendría que comprobar si el main es más viejo que los fuentes.

Reply:
Por lo que he leído, no hay un comando que compruebe la sintaxis per se en Go.

Le llamado main, pero le podría poner un nombre cualquiera, simplemente es un binario, la compilación del proyecto, que es la única manera de comprobar que la sintaxis está bien si no me equivoco.

Reply:
Sí, si lo hay. Simplemente úsalo sobre los ficheros existentes para comprobar la sintaxis.